src/styles/blocks/header.scss

```diff
@@ -0,0 +1,129 @@
+.header {
+  background-size: cover;
+  background-position: right 13% center;
+  background-image: url(../images/photos/header-bg.png);
+
+  @include onDesktop {
+    background-position: center;
+  }
+
+  @include onTablet {
+    background-position: center;
+  }
+
+  &__content {
+    display: flex;
+    flex-direction: column;
+    justify-content: space-between;
+    height: 100vh;
+    padding: 29px 0 88px;
+
+    @include onDesktop {
+      padding-top: 36px;
+      padding-bottom: 200px;
+    }
+
+    @include onTablet {
+      padding-bottom: 130px;
+    }
+  }
+
+  &__buttons {
+    display: flex;
+    align-items: center;
+    gap: 30px;
+
+    @include onDesktop {
+      gap: 40px;
+    }
+  }
+
+  &__call {
+    display: block;
+    width: 18px;
+    height: 18px;
+
+    background-image: url(../images/phone.svg);
+    background-size: cover;
+    background-position: center;
+    background-repeat: no-repeat;
+    text-decoration: none;
+
+    @include onDesktop {
+      width: 24px;
+      height: 24px;
+
+      &::before {
+        content: "+1 234 555-55-55";
```

Comment:
Bad practice. To change this phone number I need to change the style file, instead of the content.

Reply:
Didn`t get a point, what should i do exactly